spec/controllers/address_controller_spec.rb

```diff
@@ -2,28 +2,23 @@
 
 require "rails_helper"
 
-RSpec.describe AddressController, :member, type: :controller do
-  let!(:current_app) do
-    SnapApplication.create!(
-      user: member,
-      street_address: "123 Fake St",
-      city: "Springfield",
-      county: "Genesee",
-      zip: "12345",
-      state: "MI",
-    )
-  end
-
+RSpec.describe AddressController, type: :controller do
   let(:step) { assigns(:step) }
 
   describe "#edit" do
     it "assigns the correct step" do
+      session[:snap_application_id] = current_app.id
```

Comment:
I'm not convinced myself, but this might be a place (maybe the only situation) worth considering moving this into a `before` block. Dunno. What do you think?